edunext_openedx_extensions/edunext/management/commands/change_domain.py

```diff
@@ -0,0 +1,83 @@
+"""
+This module contains the command class to change
+the subdomain from prod domains to stage versions.
+"""
+import json
+
+from django.core.management.base import BaseCommand
+
+from edunext_openedx_extensions.ednx_microsites.models import Microsite
+
+
+class Command(BaseCommand):
+
+    """
+    This class contains the methods to change
+    microsite prod domains to a stage versions.
+    """
+    # pylint: disable=unused-argument
+    help = """This function will iterate over all microsites objects
+            to change microsite prod domains to a stage versions."""
+
+    def add_arguments(self, parser):
+        """
+        Add optional domain from the line command
+        """
+        parser.add_argument('suffix_domain', type=str,
+                            nargs='?', default=u"stage.edunext.co")
+
+    def handle(self, *args, **options):
+        """
+        This method will iterate over all microsites
+        objects to change microsite prod domains to a stage versions.
+        """
+
+        # pylint: disable=attribute-defined-outside-init
+        self.suffix_stage_domain = options['suffix_domain']
```

Comment:
instead of ignoring the pylint here, we can define the atribute `suffix_stage_domain` after line #19 as an empty string.